hazelcast/src/main/java/com/hazelcast/map/impl/LazyMapEntry.java

```diff
@@ -142,5 +150,16 @@ public int hashCode() {
     public String toString() {
         return getKey() + "=" + getValue();
     }
+
+    private void readObject(java.io.ObjectInputStream s) throws IOException, ClassNotFoundException {
+        keyObject = s.readObject();
+        valueObject = s.readObject();
+
+    }
+
+    private void writeObject(java.io.ObjectOutputStream s) throws IOException {
```

Comment:
What about calling `defaultWriteObject` method here? According to Effective Java it is a recommended practice. http://allenlsy.com/NOTES-of-Effective-Java-10/